src/ontology/cl-edit.owl

```diff
@@ -32370,6 +32372,15 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (substantia nigra dopaminergic neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32826893") Annotation(oboInOwl:hasDbXref "PMID:35308118") obo:IAO_0000115 obo:CL_4042023 "A midbrain dopaminergic neuron that has its soma located in a substantia nigra. This dopaminergic neuron type is metabolically active and it is involved in the regulation of movement, cognition, motivation and reward. Neurodegeneration of this dopaminergic neuronal type causes loss in fine motor control in Parkinson's Disease.")
```

Comment:
This dopaminergic neuron type is metabolically active
-->
This dopaminergic neuron type is highly metabolically active